templates/index.html

```diff
@@ -43,7 +43,11 @@ <h2 class="form-signin-heading">Please sign in</h2>
           </label>
         </div>
         <button class="btn btn-lg btn-primary btn-block" type="submit">Sign in</button>
+        <a class="btn btn-medium btn-link btn-block" href="{{ url_for('new_user_request') }}">New User?</a>
       </form>
+    {% for message in get_flashed_messages() %}
+      <div class="alert alert-success">{{ message }}</div>
```

Comment:
Place to let the user know their signup was successful. Later we can write functionality to send them an email.